tests/test_energies.py

```diff
@@ -116,5 +116,11 @@ def testNIST(self):
                 self.assertLess(max_diff / math.sqrt(natoms), self.tolerance)
 
 
+class TestEnergiesEnergyShifterJIT(TestEnergies):
+    def setUp(self):
+        super().setUp()
+        self.energy_shifter = torch.jit.script(self.energy_shifter)
```

Comment:
After this, should we manually do
```python
self.nn = torch.nn.Sequential(nnp, self.energy_shifter)
self.model = torch.nn.Sequential(self.aev_computer, nnp, self.energy_shifter)
```
again?

Reply:
Yes, fixed in 6722715